sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -785,6 +784,51 @@ def test_warm_start():
     assert_true(h.converged_)
 
 
+@ignore_warnings(category=ConvergenceWarning)
+def test_convergence_detected_with_warm_start():
+    # We check that convergence is detected when warm_start=True
+    rng = np.random.RandomState(0)
+    rand_data = RandomData(rng)
+    n_components = rand_data.n_components
+    X = rand_data.X['full']
+
+    for max_iter in (1, 2, 50):
+        gmm = GaussianMixture(n_components=n_components, warm_start=True,
+                              max_iter=max_iter, random_state=rng)
+        for _ in range(100):
+            gmm.fit(X)
+            if gmm.converged_:
+                break
+        assert_true(gmm.converged_)
```

Comment:
Minor: since moving to pytest, we're trying to avoid such assert_* functions. Use a bare assert instead.

Reply:
Okay, fixing this now.